packages/web_pointer_interceptor/README.md

```diff
@@ -0,0 +1,29 @@
+# web_pointer_interceptor
+
+A small Flutter widget to prevent clicks (in web) from being intercepted by underlying HtmlElement views.
```

Comment:
nits:

- remove "small Flutter"
- s/HtmlElement view/HtmlElementView/ (also would be useful to link to https://api.flutter.dev/flutter/widgets/HtmlElementView-class.html)